hourly/models.py

```diff
@@ -0,0 +1,328 @@
+########################################################################################
+# Plataforma para la Iniciativa Regional de Monitoreo Hidrológico de Ecosistemas Andinos
+# (iMHEA)basada en los desarrollos realizados por:
+#     1) FONDO PARA LA PROTECCIÓN DEL AGUA (FONAG), Ecuador.
+#           Contacto: info@fonag.org.ec
+#     2) EMPRESA PÚBLICA METROPOLITANA DE AGUA POTABLE Y SANEAMIENTO DE QUITO (EPMAPS),
+#           Ecuador.
+#           Contacto: paramh2o@aguaquito.gob.ec
+#
+#  IMPORTANTE: Mantener o incluir esta cabecera con la mención de las instituciones
+#  creadoras, ya sea en uso total o parcial del código.
+########################################################################################
+from __future__ import unicode_literals
+
+from typing import List, Type
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.urls import reverse
+from timescale.db.models.models import TimescaleModel
+
+from station.models import Station
+
+HOURLYS: List[str] = []
+"""Available hourly variables."""
+
+
+# TODO check if PolarWind is needed in HOurly
```

Comment:
To check with whom? If unsure, just delete these code and open an issue with it.